.github/workflows/integration_test.yaml

```diff
@@ -1,15 +1,16 @@
 name: Integration tests
-
 on:
   pull_request:
   workflow_call:
-
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: arjun11-malik/operator-workflows/.github/workflows/integration_test.yaml@main
```

Comment:
Similar to other PRs, just add the TODO: comment saying we need to change this for operator-workflows when https://github.com/canonical/operator-workflows/pull/828 gets merged and please add a tracking issue for the change.